docs/pages/guides/spend-permissions/quick-start.mdx

```diff
@@ -79,7 +79,13 @@ export async function getSpenderWalletClient() {
 
 ### Configure the Smart Wallet URL
 
-In `app/providers.tsx`, update the chain configuration to use Base Sepolia testnet by replacing all instances of `base` with `baseSepolia` in this file (including the import).
+In `app/providers.tsx`, update the value of `keysUrl` based on your environment:
+
+- For testnets: `"https://keys-dev.coinbase.com/connect"`
+- For mainnets: Leave `keysUrl` undefined (defaults to keys.coinbase.com)
+
+We also want to point our chain id to Base Sepolia testnet by setting replacing all instances of `base`
+with `baseSepolia` in this file (including the import).
```

Comment:
should this specify that we only want to swap out to `baseSepolia` in the case that we're wanting to use a testnet and have also specified keys-dev instead of keys?  This makes me feel like I'm not sure which combination of URL and chain I should be using.  Maybe the bit about sepolia could go directly with.under the "For testnets" line, and the "For mainnets" line could include something like: "the project template will default to `base`"

Reply:
updated